src/test/java/com/triviamachine/triviamachine/TriviaMachineApplicationTests.java

```diff
@@ -0,0 +1,340 @@
+package com.triviamachine.triviamachine;
+
+import com.triviamachine.triviamachine.controllers.AdminPanelController;
+import com.triviamachine.triviamachine.controllers.ApplicaitonController;
+import com.triviamachine.triviamachine.controllers.VoteController;
+import com.triviamachine.triviamachine.database.*;
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+
+import java.text.DateFormat;
+import java.util.Date;
+
+import static org.junit.Assert.fail;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TriviaMachineApplicationTests {
+
+    private MockMvc mockMvc;
+
+    @Autowired
+    public WebApplicationContext webApplicationContext;
+
+    @Autowired
+    AdminPanelController adminPanelController;
```

Comment:
adminPanelController, applicaitonController (🤣 the name is so consistent), voteController, adminUserRepository all are unused.